### ISSUE TITLE
This project is only working on MAC?

### ISSUE THREAD

**Issue by metafory** [steveruizok/telestrator#16]
Not sure if this can work on a Windows PC?

Got this ... 

Cannot build electron packages:
Error: Command failed with exit code 1: electron-builder --mac --x64 --arm64

**Comment by steveruizok** [comment]
Yes, afaik this only works on mac. Not sure if it _has_ to be that way, though.
